components/text/gawk/manifests/sample-manifest.p5m

```diff
@@ -23,8 +23,8 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 link path=usr/bin/awk target=gawk
-hardlink path=usr/bin/gawk target=gawk-$(COMPONENT_VERSION)
-file path=usr/bin/gawk-$(COMPONENT_VERSION)
+file path=usr/bin/gawk
+hardlink path=usr/bin/gawk-$(COMPONENT_VERSION) target=gawk
 file path=usr/etc/profile.d/gawk.csh
```

Comment:
Please remove the sample-manifest from your PR. The values have been simply swapped and I bet if you run gmake sample-manifest several times you will get both variants.

Reply:
Removed, thanks.